rising_world/java/egg-rising-world-java.json

```diff
@@ -0,0 +1,202 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PANEL",
+    "meta": {
+        "version": "PLCN_v1",
+        "update_url": null
+    },
+    "exported_at": "2024-12-26T20:49:48+00:00",
+    "name": "Rising World Java",
+    "author": "eagled2@zoho.com",
```

Comment:
please respect MIT Licence. Author cant be changed. You only imported the egg. My egg is licence here under MIT. With the next commit under AGPL